fuse/dl/models/model_wrapper.py

```diff
@@ -31,10 +31,10 @@ class ModelWrapper(torch.nn.Module):
 
     def __init__(self,
                  model: torch.nn.Module = None,
-                 model_inputs: Sequence[str] = ('data.input.input_0.tensor',),
+                 model_inputs: Sequence[str] = None, #('data.input.input_0.tensor',),
```

Comment:
Lets remove the default value.
Move the outputs to be the second argument (and also remove the default value)

Reply:
done (I assume you meant 3rd argument, write here if you prefer output to be before input)

Reply:
also prevent positional args, and added example values to the constructor docstring